File: Codechef/Helping Chef/Helping chef.cpp
#include <iostream>
using namespace std;

int main()
{
	//declaration of no of test cases
	int t;

    //taking input of no of test cases
	cin>>t;

    //loop for running all the test cases
	while(t--)
	{
        //declaration of n
	    int n;

        //inputting value of n
	    cin>>n;

        //checking if n is less than 10 or not and printing the output accordingly
	    if(n<10)
	    {
	        cout<<"Thanks for helping Chef!"<<endl;
	    }
	    else
	    {
	        cout<<"-1"<<endl;
	    }
	}
	return 0;
}

File: Codechef/Helping Chef/Helping_Chef.py
// Taking input value of n
n=int(input())
// List l
l=[] 
//loop from 0 to n-1
for i in range(n):
    a=int(input()) 
    //checking if n is less than 10 or not and add the output accordingly
    if a<10:
        l.append("Thanks for helping Chef!")   
    else:
        l.append(-1)
for i in l:
    print(i) // print the output stored in list 
